app/serializers/site_serializer.rb

```diff
@@ -260,6 +261,17 @@ def include_anonymous_default_sidebar_tags?
       SiteSetting.default_sidebar_tags.present?
   end
 
+  def anonymous_sidebar_sections
+    SidebarSection
+      .includes(sidebar_section_links: :linkable)
```

Comment:
Now that we have a strategy to test for N+1 queries problem. I think it will be useful to preemptively add a test for it here since the site serializer is used on every page load.